src/main/kotlin/lotto/dto/InputPaymentRequestDto.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.dto
+
+data class InputPaymentRequestDto(
+    val payment: Int
+) {
+
+    companion object {
+
+        const val CANNOT_CONVERT_INT = "정수로 변환할 수 업습니다."
+        fun convertInt(inputPayment: String): InputPaymentRequestDto {
```

Comment:
```suggestion
        const val CANNOT_CONVERT_INT = "정수로 변환할 수 업습니다."
        
        fun convertInt(inputPayment: String): InputPaymentRequestDto {
```
불필요한 공백은 제거해주시고, 변수와 함수 사이에 개행을 통해서 구분하면 좋을 것 같아요.